src/backend/distributed/commands/utility_hook.c

```diff
@@ -43,6 +43,7 @@
 #include "foreign/foreign.h"
 #include "lib/stringinfo.h"
 #include "nodes/makefuncs.h"
+#include "nodes/nodes.h"
```

Comment:
Do we really need to include this?

Reply:
Tested and see that we don't need actually